static/elements/chromedash-approvals-dialog.js

```diff
@@ -77,37 +76,37 @@ class ChromedashApprovalsDialog extends LitElement {
     return [
       ...SHARED_STYLES,
       css`
-        .loading {
+        :host .loading {
```

Comment:
Is the :host needed?  I thought everything in the shadow DOM is under :host.

Reply:
You're right.  I was confused,thinking my styles were starting to work once I added the :host context.  But it must have been for some other reason.